css/colourpicker.css

```diff
@@ -36,36 +42,59 @@
   left: 0;
   top: auto;
 }
+
+
 .palette-square .colourpicker-panel {
-  width: 173px;
+  width: 174px;
   height: 152px;
 }
+
+.palette-square.colourpicker-with-alpha .colourpicker-panel {
+  width: 196px;
+}
+
 .palette-limited .colourpicker-panel {
-  width: 181px;
+  width: 196px;
   height: auto;
-  padding: 5px;
+  padding-left: 0px;
+  padding-right: 0px;
+  padding-top: 1px;
+  padding-bottom: 4px;
   background-color: #F8F8F8;
 }
-.colourpicker .cp-list-row {
+
+/*.colourpicker .cp-list-row {
```

Comment:
instead of commenting out, just delete it. That's what git is for :) Codebase will become messy quick if changes are commented instead of removed